presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -362,18 +361,7 @@ protected Node visitShowPartitions(ShowPartitions showPartitions, Void context)
             }
 
             QualifiedObjectName partitionsTable = new QualifiedObjectName(table.getCatalogName(), table.getSchemaName(), table.getObjectName() + "$partitions");
-            if (!metadata.getTableHandle(session, partitionsTable).isPresent()) {
-                throw new SemanticException(NOT_SUPPORTED, showPartitions, "Table does not have partition columns: %s", table);
-            }
-
-            return simpleQuery(
-                    selectList(new AllColumns()),
-                    new Table(createQualifiedName(partitionsTable)),
-                    showPartitions.getWhere(),
-                    Optional.empty(),
-                    Optional.empty(),
-                    orderBy(showPartitions.getOrderBy()),
-                    showPartitions.getLimit());
+            throw new SemanticException(NOT_SUPPORTED, showPartitions, "SHOW PARTITIONS no longer exists. Use this instead: SELECT * FROM \"%s\"", partitionsTable);
```

Comment:
Commit says:
> Deprecate SHOW PARTITIONS

Reply:
I've read https://github.com/prestodb/presto/pull/10525 now. Fine, but choose different cmt msg, as deprecated functionality usually is one that works (until removed).
Perhaps `Remove support for SHOW PARTITIONS statement` and then later `Remove remainings of SHOW PARTITIONS statement`?

Reply:
Agreed, I will update the commit messages.